### PR TITLE
Prepare 7.6.4 release

### PR DESCRIPTION
**Pull request by ov7a**
### Reviewing cheatsheet

Before merging the PR, comments starting with 
- ❌ ❓**must** be fixed
- 🤔 💅 **should** be fixed
- 💭 **may** be fixed
- 🎉 celebrate happy things

### CODE REVIEW

File: subprojects/docs/src/docs/release/notes.md [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
 The Gradle team is excited to announce Gradle @version@.
 
-This is the third patch release for Gradle 7.6.
+This is the fourth patch release for Gradle 7.6.
+
+This release addresses security vulnerabilities:
+
+* [Apache Ant insecure temporary file vulnerability](https://github.com/gradle/gradle/security/advisories/GHSA-j45w-qrgf-25vm)
+* [Guava insecure temporary directory](https://github.com/gradle/gradle/issues/25945)
+* [Ivy Improper Restriction of XML External Entity Reference](https://nvd.nist.gov/vuln/detail/CVE-2022-46751) and related Gradle issue [#27855](https://github.com/gradle/gradle/issues/24795)
```
Comment (ljacomet) [inline subprojects/docs/src/docs/release/notes.md:9]: ❌ The second link label and issue do not match: 27855 vs 24795

**Review by ljacomet — CHANGES_REQUESTED** [review]
Looks good overall, but see one mismatch below in release notes

File: subprojects/docs/src/docs/release/notes.md (hunk 1 above)
Comment (ov7a) [reply subprojects/docs/src/docs/release/notes.md]: Thanks! No idea where it came from.

**Comment by ov7a** [comment]
@bot-gradle merge

**Pull request merged**